setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+dependencies = ['pytest', 'pytest-cov']
```

Comment:
Why are these required for installation of this package? Aren't these only used in testing?